index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins:400,700,900' rel='stylesheet'>
+  </head>
+  <body>
+    <div id="container">
+      <header>
+
+        <div id="app-bar-compact">
+          <div id="toolbar">
+            <div id="div-button">
+
+              <nav class="menu">
+
+                <button id="icon-button-button">
+                  <span id="union">
+                    <span class="vector">
+                    </span>
+                    <span class="vector">
+                    </span>
+                    <span class="vector">
+                    </span>
+                  </span>
+                </button>
+
+              </nav>
+
+            </div>
```

Comment:
- [ OPTIONAL] The content of your toolbar is supposed to be aligned with flex, please use flexbox, it will help you to improve the responsiveness of your app.